### DISCUSSION TITLE
Need to add label name of bubble chart data series as x-axis label name

### DISCUSSION THREAD

**Discussion by rajhen96** [nhn/tui.chart#610] [Q&A]
Need to add the bubble chart data series label value as x-axis label.

Eg;
Having series as data:[ { x:20, y:20, r:20, label:'XXXX' } ].Need to set 'XXXX' label value as x-axis label in bubble chart. Kindly provide the solution ASAP.

**Comment by jung-han** [comment]
@rajhen96 

Use Axes formatter.(https://github.com/nhn/tui.chart/blob/main/docs/en/common-axes.md#formatter)
Based on value, you can find and change the value of the series.

**Comment by rajhen96** [comment]
@jung-han I tried like the way mentioned in the document
Code snippet used for formatting labels of x-axis in bubble chart: 
x-axis:
{
label:
{
formatter:(value)=>
{
return `${value.x} +','+ ${value.label}`
},},},

Please help me on how to get the series data in the value.

**Comment by jung-han** [comment]
@rajhen96 
In the axis formatter, the axis value is `string` value.
Since there can be multiple values of x in series, it does not seem necessary to pass the values of the series to the value.
After converting the `value` to number in the axis formatter, it would be better to find the x in the series data used when creating the chart and use it.
